src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -637,6 +637,7 @@ public float RowSpan
             set
             {
                 SetValue(RowSpanProperty, value);
+                SetValue(GridLayout.RowSpanProperty, (int)value);
```

Comment:
I am not sure whether this is OK because I don't understand Layout very well,
but with quick glance, it seems to set same value for two times, even when GridLayout is not used.
I don't know this is right or not so I think other member's review is required.

Reply:
I have question, too. 
As fas as I understand,
`SetValue(GridLayout.RowSpanProperty, (int)value)` will eventually call `view?.Layout?.RequestLayout();`.
If this code was all what you wanted, how about just adding `Layout?.RequestLayout()` here, instead `SetValue`?

Reply:
`Layout?.RequestLayout()` is not related with this issue.
GridLayout will try to get `GridLayout.RowSpanProperty` value of children on `GridLayout.OnMeasure()`. howerver, this value is always 1(default value) because of the reason in the commit message.

Reply:
Thank you for the explanation.
I am not familiar with attached bindable properties, so I have search for it,
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/attached-properties#consume-an-attached-property
```
<Label Text="Label Shadow Effect" local:ShadowEffect.HasShadow="true" />
```
In Xamarin, above example works.

In the description you wrote,
> Since xaml parser find property setter by property name, the parser will find
View.ColumnSpan even if you use "GridLayout.ColumnSpan" in xaml.

I wonder why NUI is different from Xamarin despite it uses same xaml parser.

Reply:
> I wonder why NUI is different from Xamarin despite it uses same xaml parser.

The order of setting property is different.
Please check NUI `XamlG`(https://github.com/Samsung/TizenFX/pull/1639) and Xamarin.Forms `XmalG` (https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Xaml/ApplyPropertiesVisitor.cs#L405) 

Xamarin.Forms `XamlG` try to set bindable property first, but NUI `XamlG` try to set normal property.
If you test without https://github.com/Samsung/TizenFX/pull/1639 patch, `GridLayout.ColumnSpan` will work properly.


BTW, NUI `XamlC` is the same to Xamarin.Forms `XamlC`

NUI `XamlC` :
- https://github.com/nui-dali/Tizen.NUI.XamlBuild/blob/master/src/internal/Xaml/ApplyPropertiesVisitor.cs#L346

Xamarin.Forms `XamlC` :
-  https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs#L943

So, if you add below code in `YourDemoApp.csproj`(that means using XamlC instead of XamlG), `GridLayout.ColumnSpan` will work properly.
```xaml
  <PropertyGroup>
    <NeedInjection>True</NeedInjection>
  </PropertyGroup>
```

> Is it because there are duplicate property names between View and GridLayout?

Yes, if property names are different, `TrySetProperty` will return false and `TrySetValue` try to assing attached property.